library/compiler-builtins/compiler-builtins/Cargo.toml

```diff
@@ -19,6 +19,7 @@ links = "compiler-rt"
 bench = false
 doctest = false
 test = false
+doc = false
```

Comment:
Would you mind adding a comment to these so they doesn't accidentally get removed? After that, r=me